src/NLog/SetupExtensionsBuilderExtensions.cs

```diff
@@ -133,7 +134,7 @@ public static ISetupExtensionsBuilder RegisterLayoutRenderer(this ISetupExtensio
         /// <param name="layoutMethod">Callback that returns the value for the layout renderer.</param>
         public static ISetupExtensionsBuilder RegisterLayoutRenderer(this ISetupExtensionsBuilder setupBuilder, string name, Func<LogEventInfo, object> layoutMethod)
         {
-            return RegisterLayoutRenderer(setupBuilder, name, (info, configuration) => layoutMethod(info));
+            return RegisterLayoutRenderer(setupBuilder, name, (info, configuration) => layoutMethod(info), LayoutRenderOptions.None);
```

Comment:
If you remove `LayoutRenderOptions.None` from this method, then I think you will get better code-coverage by the unit-tests.